src/platform/unittest/TlsTest.cpp

```diff
@@ -1835,6 +1930,25 @@ TEST_F(TlsTest, ClientCertificateDeferValidation)
     DoHandshake(ServerContext, ClientContext);
 }
 
+#ifndef _WIN32
```

Comment:
Generally, we want more specific flags such as the `QUIC_ENABLE_ANON_CLIENT_AUTH_TESTS` below. Not just using `_WIN32`.